app/models/disc_evidence_coversheet.rb

```diff
@@ -34,7 +34,7 @@ def current_date
   end
 
   def fee_scheme
-    @fee_scheme || claim.fee_scheme.name.upcase
+    @fee_scheme || claim&.fee_scheme&.name
```

Comment:
I think you still need the upcase on this. As it is displayed like that in the PDF

Reply:
Actually looks like fee schemes names are upper case in any event so the checks like `fee_scheme.eql?('AGFS')`on line37 and 41 in this file would work. Could make those checks case insensitive for safety e.g (`fee_scheme.match?(/AGFS/i)` or `fee_scheme.upcase.eql?('AGFS')`)